src/app/feature-flag.module.ts

```diff
@@ -21,7 +22,7 @@ import { FeatureWarningPageComponent } from './warning-page/feature-warning-page
     FeatureWarningPageComponent,
     FeatureFlagHomeComponent
   ],
-  providers: [FeatureFlagMapping]
+  providers: [FeatureFlagMapping, FeatureFlagResolver]
```

Comment:
Hmm. Should this be in `providers`, or just `exports`? I think I'd rather have this module simply export the resolver, and consumers decide where and when to inject it into their DI hierarchy. FWIW the fabric8-ui app.module already has the resolver in its own `providers` array at the top level of the application.